utils/sparse.py

```diff
@@ -200,20 +200,41 @@ def qat_active(self, epoch):
 
         return qat_start < epoch + 1
 
-    def reset_best(self, epoch):
+    def get_best_save_names(self, epoch):
+        # return a list of the names to save a "best" model under
+
+        save_names = ["best_overall"]
```

Comment:
As discussed online: Recommend removing `best_overall`. A user may not understand what best_overall could be, and may not know that its not pruned or quantized. We should have breaking changes with what we output vs yolo so there is no ambiguity about what is what